cucumber-expressions/ruby/lib/cucumber/cucumber_expressions/cucumber_expression_generator.rb

```diff
@@ -85,19 +83,17 @@ def create_parameter_type_matchers(text)
       end
 
       def create_parameter_type_matchers2(parameter_type, text)
-        result = []
         regexps = parameter_type.regexps
-        regexps.each do |regexp|
+        regexps.map do |regexp|
           regexp = Regexp.new("(#{regexp})")
-          result.push(ParameterTypeMatcher.new(parameter_type, regexp, text, 0))
+          ParameterTypeMatcher.new(parameter_type, regexp, text, 0)
         end
-        result
       end
 
       def escape(s)
         s.gsub(/%/, '%%')
         .gsub(/\(/, '\\(')
-        .gsub(/\{/, '\\{')
+        .gsub(/{/, '\\{')
```

Comment:
Is `{` not a reserved character?

Reply:
It is, but whether it needs to be escaped or not depends on the context. IDEA told me escaping was redundant here, and I trust it :-)